py-polars/tests/unit/test_predicates.py

```diff
@@ -229,3 +229,18 @@ def test_fast_path_boolean_filter_predicates() -> None:
     df = pl.DataFrame({"colx": ["aa", "bb", "cc", "dd"]})
     assert_frame_equal(df.filter(False), pl.DataFrame(schema={"colx": pl.Utf8}))
     assert_frame_equal(df.filter(True), df)
+
+
+def test_predicate_pushdown_boundary() -> None:
+    df = pl.DataFrame({"x": [1, 2, 4]})
+
+    # tests: 12102
```

Comment:
Can you put this number at the end of the test name? E.g.: `test_predicate_pushdown_boundary_12102`


EDIT: I shall do this.